src/auth/Fingerprint.cpp

```diff
@@ -153,41 +158,61 @@ void CFingerprint::handleVerifyStatus(const std::string& result, bool done) {
     Debug::log(LOG, "fprint: handling status {}", result);
     auto matchResult   = s_mapStringToTestType[result];
     bool authenticated = false;
+    bool retry         = false;
     if (m_sDBUSState.sleeping && matchResult != MATCH_DISCONNECTED)
         return;
     switch (matchResult) {
         case MATCH_INVALID: Debug::log(WARN, "fprint: unknown status: {}", result); break;
         case MATCH_NO_MATCH:
             stopVerify();
             if (m_sDBUSState.retries >= 3) {
-                m_sDBUSState.message = "Fingerprint auth disabled (too many failed attempts)";
+                m_sPrompt         = "";
+                m_sFailureReason = "Fingerprint auth disabled (too many failed attempts)";
             } else {
                 done = false;
                 startVerify(true);
-                m_sDBUSState.message = "Fingerprint not matched";
             }
             break;
         case MATCH_UNKNOWN_ERROR:
             stopVerify();
-            m_sDBUSState.message = "Fingerprint auth disabled (unknown error)";
+            m_sPrompt         = "";
+            m_sFailureReason = "Fingerprint auth disabled (unknown error)";
             break;
         case MATCH_MATCHED:
             stopVerify();
-            m_sDBUSState.message = "";
-            authenticated        = true;
+            m_sPrompt         = "Fingerprint matched!";
```

Comment:
Wait that is not a "prompt"?
I prompt should indicate what we expect the user to do. `Password: `, `TOTP: ` or in this context smth like `Put your finger on the sensor`.

Reply:
fair enough